stake-pool/program/src/processor.rs

```diff
@@ -484,13 +482,14 @@ impl Processor {
     }
 
     /// Processes `Initialize` instruction.
+    #[inline(never)] // needed due to stack size violation
     fn process_initialize(
         program_id: &Pubkey,
         accounts: &[AccountInfo],
-        fee: Fee,
+        epoch_fee: Fee,
         withdrawal_fee: Fee,
-        stake_deposit_fee: Fee,
-        stake_referral_fee: u8,
+        deposit_fee: Fee,
```

Comment:
The idea is that this is changing the behavior, so that the deposit fee set at init is actually for both stake and sol deposit.  it seemed unclear to me that only the stake deposit fee is set at init, but not the sol deposit fee.  this way all of them are set the same on init, to minimize surprise to a pool creator.  And I didn't want to have to specify each of the fees separately on init, just to keep things simpler.